lib/hashie/extensions/dash/indifferent_access.rb

```diff
@@ -11,6 +11,7 @@ module ClassMethods
           # Check to see if the specified property has already been
           # defined.
           def property?(name)
+            name = defined?(translations) && translation_exists?(name) ? translations[:"#{name}"] : name
```

Comment:
I would find a better way to check whether we have `translations`, ie. that `Hashie::Extensions::Dash::PropertyTranslation` has been included in the class. Or maybe the latter should re-prepend `property?`.